Source/Utils/CoarsenIO.H

```diff
@@ -1,11 +1,11 @@
-#ifndef WARPX_AVERAGE_H_
-#define WARPX_AVERAGE_H_
+#ifndef WARPX_COARSEN_IO_H_
+#define WARPX_COARSEN_IO_H_
 
 #include "WarpX.H"
 
-namespace Average{
+using namespace amrex;
 
-    using namespace amrex;
+namespace CoarsenIO{
```

Comment:
`using namespace amrex;` freely in a `.H` file is dangerous because it would implicitly be included in all files that include this `.H` file. So it must be either in a cpp file or inside a namespace, or in the body of a function etc.
```suggestion
namespace CoarsenIO{
using namespace amex;
```